src/main/java/liqp/filters/Newline_To_Br.java

```diff
@@ -0,0 +1,15 @@
+package liqp.filters;
+
+class Newline_To_Br extends Filter {
+
+    /*
+     * newline_to_br(input)
+     *
+     * Add <br /> tags in front of all newlines in input string
+     */
+    @Override
+    public Object apply(Object value, Object... params) {
+
+        return super.asString(value).replaceAll("[\n]", "<br />\n");
```

Comment:
Cool, thanks!

Perhaps nice to include `\r` for users on WIndows?

    replaceAll("\r?\n|\r", "<br />$0")

Note that `$0` contains the entire regex match, which is either `\r\n` (Windows), `\n` (Linux/Mac) or only `\r` (old Macs)